mobile/src/main/java/org/openhab/habdroid/ui/OpenHABPreferencesActivity.java

```diff
@@ -224,17 +243,86 @@ public boolean onPreferenceClick(Preference preference) {
                 }
             });
 
-            //fullscreen is not supoorted in builds < 4.4
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
-                final PreferenceScreen ps = getPreferenceScreen();
-                ps.removePreference(ps.findPreference(Constants.PREFERENCE_FULLSCREEN));
+            ringtonePreference.setOnPreferenceChangeListener(new OnPreferenceChangeListener() {
+                @Override
+                public boolean onPreferenceChange(Preference preference, Object newValue) {
+                    updateRingtonePreferenceSummary(preference, newValue);
+                    return true;
+                }
+            });
+
+            final PreferenceScreen ps = getPreferenceScreen();
+
+            // Fullscreen is not supoorted in Android < 4.4
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT) {
+                Preference fullscreenPreference = ps.findPreference(Constants.PREFERENCE_FULLSCREEN);
+                fullscreenPreference.setEnabled(false);
+                fullscreenPreference.setSummary(R.string.settings_disabled_kitkat);
+            }
```

Comment:
I'd keep this removed. There's nothing the user can do anyway.

Reply:
Removing settings didn't work for me at all and I think it's better to give the user feedback why it's not supported.

Reply:
> I think it's better to give the user feedback why it's not supported.

And what is the user supposed to do with that information? I'm all for informing the user, but if he can't do anything about it, it IMHO just adds clutter.